Conceptpower+Spring/src/main/java/edu/asu/conceptpower/rest/msg/json/JsonConceptMessage.java

```diff
@@ -40,32 +41,34 @@ private ConceptEntryMessage getConceptMessage(ConceptEntry entry, ConceptType ty
 
         ConceptEntryMessage json = new ConceptEntryMessage();
 
-        json.setId(entry.getId());
+        json.setId(StringEscapeUtils.escapeJson(entry.getId()));
         json.setConceptUri(uriCreator.getURI(entry));
-        json.setLemma(entry.getWord());
-        json.setPos(entry.getPos());
-        json.setDescription(entry.getDescription());
-        json.setConceptList(entry.getConceptList());
+        json.setLemma(StringEscapeUtils.escapeJson(entry.getWord()));
+        json.setPos(StringEscapeUtils.escapeJson(entry.getPos()));
```

Comment:
pos should never have any special characters in it (it's either noun, verb, adjective, or adverb).